src/OpenLoco/Town.cpp

```diff
@@ -55,6 +56,9 @@ namespace OpenLoco
             max_company_rating);
     }
 
+    // 0x004FF714
+    constexpr uint32_t _populations[]{ 0, 150, 500, 2500, 8000 };
```

Comment:
populationLimits?

Reply:
These determine what is a Hamlet, Village (from 150), Town, City or Metropolis. It changes the label on the town window and I think it also affects what types of buildings are built (houses or skyscrapers).

Reply:
Thanks, that concurs with Duncan's implementation. I do think we should rename the array to something more descriptive, however. `populationSizeThresholds` or `populationLimits`, perhaps?

Reply:
Yeah i was thinking similar. I wasn't 100% sure when i wrote it if it was population as it's operating on a different variable than population. But now that I've tested it i can confirm that it is population related.

Reply:
values look correct to me